### PR TITLE
BC1-58-fix:fix search box on top bar and add js file for search box

### PR DESCRIPTION
**Pull request by sinaalimi**

### CODE REVIEW

File: product-details/js/main.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      // $(this).removeClass("icon-circle-down");
+      $(this).toggleClass("icon-circle-down");
+      $(this).toggleClass("icon-circle-up");
+    });
+
+    $(".section-responsive").click(function(){
+      $(".menu").toggle('slide');
+      $(".menu").css('display', 'block');
+      // $(this).removeClass("icon-menu5");
+      // $(this).addClass("icon-close dbl-click");
```
Comment (MichaelAndish) [inline product-details/js/main.js:32]: if you don't use it, just remove the comments

File: product-details/scss/layout/nav-bar.scss [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+            display: flex;
+            align-items: center;
+
+            @include mini_laptop {
+                display: block;
+                width: 100vw;
+                background-color: $white;
+            }
+
+            .Circle {
```
Comment (MichaelAndish) [inline product-details/scss/layout/nav-bar.scss:51]: why it's capitalcase? `.Circle`

File: product-details/scss/layout/top-bar.scss [hunk 3]
```diff
@@ -0,0 +1,341 @@
+/*------------------------------------*\
+  #TOP BAR
+\*------------------------------------*/
+.top-bar {
+  width: 100vw;
+  background: $white;
+  // color: $white;
```
Comment (MichaelAndish) [inline product-details/scss/layout/top-bar.scss:7]: if you don't use it, remove the comment

File: product-details/scss/layout/top-bar.scss [hunk 4]
```diff
@@ -0,0 +1,341 @@
...
+          @include mini_laptop {
+            display: inline;
+            color: $secondColorWeprodev;
+            max-width: 64em;
+
+          }
+
+
+          li {
+            list-style: none;
```
Comment (MichaelAndish) [inline product-details/scss/layout/top-bar.scss:53]: why we have 2 types of mixin name? retinaMobile is camelCase and  mini_laptop is snake_case ?!

File: product-details/scss/layout/top-bar.scss [hunk 5]
```diff
@@ -0,0 +1,341 @@
...
+        }
+
+        .button2 {
+          background-color: $mainColorWeprodev;
+          padding: 0.9em 2em;
+          color: $black;
+          border-color: $mainColorWeprodev;
+          border-radius: 4px;
+          border-width: 1px;
+
```
Comment (MichaelAndish) [inline product-details/scss/layout/top-bar.scss:310]: Maybe you can re-use the shared component?!

**Review by MichaelAndish — CHANGES_REQUESTED** [review]

File: product-details/index.html [hunk 6]
```diff
@@ -1,35 +1,154 @@
...
+  <script src="js/slider.js"></script>
+  <script>
+    window.onscroll = function () {
+      myFunction();
+    };
+
+    var navbar = document.getElementById("header");
+    var sticky = navbar.offsetTop;
+
+    function myFunction() {
```
Comment (abdolmajed3) [inline product-details/index.html:144]: i think its better to change function name.

>  Prefer descriptive variable and function names.

File: product-details/js/main.js [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+
+  });
+
+window.onscroll = function() {myFunction()};
+
+var navbar = document.getElementById("header");
+var sticky = navbar.offsetTop;
+
+function myFunction() {
```
Comment (abdolmajed3) [inline product-details/js/main.js:73]: > - Prefer descriptive variable and function names.

**Comment by MichaelAndish** [comment]
@sinaalimi you have icomoon in 2 places, why?! shared folder and inside the project

**Pull request merged**